app/src/main/cpp/skyline/services/am/storage/IStorage.h

```diff
@@ -29,15 +40,15 @@ namespace skyline::service::am {
          */
         template<typename ValueType>
         void Push(const ValueType &value) {
-            if (offset + sizeof(ValueType) > content.size())
+            if (offset + sizeof(ValueType) > this->GetSpan().size())
                 throw exception("The supplied value cannot fit into the IStorage");
 
-            std::memcpy(content.data() + offset, reinterpret_cast<const u8 *>(&value), sizeof(ValueType));
+            std::memcpy(this->GetSpan().data() + offset, reinterpret_cast<const u8 *>(&value), sizeof(ValueType));
```

Comment:
Do you really want to do `GetSpan` twice, just do it once and store it in a local variable.